src/applications/Reservations.tsx

```diff
@@ -213,152 +244,215 @@ const Reservations = (): JSX.Element | null => {
+                {isCalendar ? (
+                  <Card border>
+                    {event.eventReservationUnits.map((eru) => (
+                      <div key={eru.reservationUnitId}>
+                        <Actions>
+                          <Select
+                            label={t('Reservations.weekSelectLabel')}
+                            multiselect={false}
+                            icon={<IconCalendar />}
+                            options={getWeekOptions(t, event)}
+                            value={week}
+                            onChange={(w) => {
+                              setWeek(w);
+                            }}
+                          />
+                          <Button
+                            id="b"
+                            variant="secondary"
+                            onClick={() => {
+                              setWeek(getWeekOption(new Date(), t));
+                            }}>
+                            {t('common.today')}
+                          </Button>
+                        </Actions>
+
+                        <TwoColLayout>
+                          <IconCalendar />
+                          <div>
+                            {displayDate(startDate, t)} -{' '}
+                            {displayDate(endDate, t)}
+                          </div>
+                          {weekEvents.map((reservation) => {
+                            const begin = parseDate(reservation.begin);
+                            const end = parseDate(reservation.end);
+                            return (
+                              <>
+                                <div>
+                                  {t(`common.weekDay.${begin.getDay()}`)}
+                                </div>
+                                <div>
+                                  {' '}
+                                  {t('common.time', {
+                                    date: begin,
+                                  })}{' '}
+                                  - {t('common.time', { date: end })}
+                                </div>
+                              </>
+                            );
+                          })}
+                        </TwoColLayout>
+                        {Array.from(resUnitEvents.entries()).map(
+                          ([reservationUnit, resUnitReservations]) => {
+                            return (
+                              <>
+                                <HorisontalRule />
+                                <TwoColLayout>
+                                  <IconHome />
+                                  <ReservationUnitName>
+                                    {reservationUnit.name.fi}
+                                  </ReservationUnitName>
+                                  <IconInfoCircle />
+                                  <ContactInfo>
+                                    TODO contact info from API
```

Comment:
Done later?

Reply:
yeah, different issue, will submit pr after this is merged